src/priceHistory/saga.ts

```diff
@@ -13,7 +13,12 @@ import { call, put, spawn, takeLatest } from 'typed-redux-saga'
 
 const TAG = 'priceHistory/saga'
 
-/** @beta - Exclude from Knip dep check */
+/** @beta - Exclude from Knip unused exports
```

Comment:
is this still needed?

Reply:
Removed in e988643a3f991108c7f0af6dc87381c2840a667a!